packages/appium/types/cli.ts

```diff
@@ -105,6 +111,10 @@ export interface PluginExtArgs {
   plugin?: string;
 }
 
+export interface SetupArgs {
+  setupCommand: CliCommandSetupSubcommand;
```

Comment:
Should this perhaps be `setupCommand?`? The description suggests that `appium setup` with no extra arguments is also accepted.